release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -82,6 +82,33 @@ You can now install a cluster on {ibm-cloud-name} in an environment with limited
 [id="ocp-4-15-web-console"]
 === Web console
 
+[id="ocp-4-15-developer-perspective"]
+==== Developer Perspective
+
+With this release, there are several updates to the *Developer* perspective of the web console.
+
+* Pipeline history and logs based on the data from Tekton Results are available in the dashboard without requiring PipelineRun CRs on the cluster.
+
+[id="ocp-4-15-software-supply-chain-enhancements"]
+===== Software Supply Chain Enhancements
+
+The *PipelineRun Details* page in the *Developer* or *Administrator* perspective of the web console provides an enhanced visual representation of PipelineRuns within a Project.
+
+For more information, see link:https://docs.openshift.com/pipelines/latest/about/understanding-openshift-pipelines.html[{pipelines-title}].
+
+[id="ocp-4-15-red-hat-developer-hub"]
+===== {rh-dev-hub} in the web console
+With this update, a quick start is now available for you to learn more about how to install and use the developer hub.
```

Comment:
I would make it "about installing and using". I think "about how to" is somewhat unwieldy.